src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusServiceSpec.scala

```diff
@@ -47,7 +47,7 @@ class ConsignmentStatusServiceSpec extends AnyFlatSpec with MockitoSugar with Re
 
     val response = consignmentService.getConsignmentStatus(consignmentId).futureValue
 
-    response should be(CurrentStatus(None))
+    response should be(CurrentStatus(None, None))
```

Comment:
This is just a suggestion so feel free to ignore me but do you think it's worth adding another couple of tests in here. One to check that if you only return the `transferAgreement` status then the `upload` status is `None` and vice versa.  What do you think?

Reply:
Yes, that's a good idea; I'll add these.